examples/cli/Makefile.am

```diff
@@ -40,10 +40,10 @@ soc_CPPFLAGS                                                   = \
     $(NULL)
 
 soc_LDADD                                                      = \
-    $(top_srcdir)/src/core/libopenthread.a                       \
-    $(top_srcdir)/src/cli/libopenthread-cli.a                    \
-    $(top_srcdir)/examples/platform/posix/libopenthread-posix.a  \
-    $(top_srcdir)/third_party/mbedtls/libmbedcrypto.a            \
+    $(top_builddir)/src/core/libopenthread.a                     \
+    $(top_builddir)/src/cli/libopenthread-cli.a                  \
+    $(top_builddir)/examples/platform/posix/libopenthread-posix.a \
```

Comment:
Is there a line width we are trying to keep the slashes at, or is it just at the furthest out that keeps them all inline?